lapce-core/src/syntax/util.rs

```diff
@@ -48,6 +50,8 @@ pub fn matching_char(c: char) -> Option<char> {
         ')' => '(',
         '[' => ']',
         ']' => '[',
+        '<' => '>',
+        '>' => '<',
```

Comment:
Shouldn't the `matching_bracket_general` function defined below also be updated?

Reply:
🤨 How did I miss that? Fixed. Thank you!